client/services/FeedbackGeneratorServiceSpec.js

```diff
@@ -92,4 +99,149 @@ describe('FeedbackGeneratorService', function() {
         "more efficiently?"].join(''));
     });
   });
+
+  describe('_getBuggyOutputTestFeedback', function() {
+    it(['should return the next hint in sequence for buggy outputs, ',
+        'provided the code has been changed'].join(''), function() {
+      var buggyOutputTestDict = {
```

Comment:
It might be worth declaring this more globally (in the 'describe', but outside the 'it') if you're going to reuse it a lot. Up to you, though.